filament/src/ShadowMapManager.cpp

```diff
@@ -458,8 +459,18 @@ ShadowMapManager::ShadowTechnique ShadowMapManager::updateCascadeShadowMaps(FEng
         // entire camera frustum, as if we only had a single cascade.
         ShadowMap& shadowMap = *mCascadeShadowMaps[0];
 
+        const auto direction = lightData.elementAt<FScene::DIRECTION>(0);
```

Comment:
Shouldn't we pick the index of the main directional light? (i.e. the one of highest intensity)

Reply:
`lightData` only store the main directional light, at index 0.  This is done in `FScene::prepare`